pkg/fakerp/fakerp.go

```diff
@@ -94,7 +92,7 @@ func CreateOrUpdate(ctx context.Context, oc *v20180930preview.OpenShiftManagedCl
 
 	// write out development files
 	log.Info("write helpers")
-	err = writeHelpers(cs, azuretemplate)
```

Comment:
revert this change since it's only used in here.

Reply:
ok, i'll revert. My thought was to have the read/write next to each other for maintainability.

Reply:
Agreed, they are all over the place, we may want to revive https://github.com/openshift/openshift-azure/issues/642